tests/test_dealer.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase
+
+from card import Card
+from player.dealer import Dealer
+
+
+class PlayerTestCase(TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
It's not a huge deal in this specific circumstance, but traditionally, we would want to call super in the `setUp*` methods. In our case, it wouldn't do anything. But if we were to make a base test class that we updated all our tests to use instead, then we would need to go in and update all of these.

```python
@classmethod
def setUpClass(cls):
    super().setUpClass()
    ...
```